docs/release.sh

```diff
@@ -116,17 +84,20 @@ invalidate_cache() {
 
 	aws configure set preview.cloudfront true
 
-	files=$(find site/ -not -name "*.md*" -type f | sed 's/site\///g')
+	# Get all the files
+	# not .md~ files
+	# not files w spaces bc they mess up the array
```

Comment:
yes we want the trailing slash for the invalidation I realized and see
updated comment for the fact you cannt invalidate a file with a space in
the name it comes back as "invalid"

On Thu, Mar 19, 2015 at 2:31 PM, Tibor Vass <notifications@github.com>
wrote:

> In docs/release.sh
> <https://github.com/docker/docker/pull/11481#discussion_r26799686>:
>
> > @@ -116,17 +84,20 @@ invalidate_cache() {
> >
> >  	aws configure set preview.cloudfront true
> >
> > -	files=$(find site/ -not -name "*.md*" -type f | sed 's/site\///g')
> > +	# Get all the files
> > +	# not .md~ files
> > +	# not files w spaces bc they mess up the array
>
> you should also remove this comment then :)
>
> Also did you intentionally remove a backslash from the sed ? used to be s/site\///g'
> and is noes/site//g`.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/11481/files#r26799686>.
>